doc/manual/source/protocols/json/schema/derivation-v3.yaml

```
@@ -0,0 +1,164 @@
+"$schema": http://json-schema.org/draft-04/schema#
+"$id": https://nix.dev/manual/nix/latest/protocols/json/schema/derivation-v3.json
+title: Derivation
+description: |
+  Experimental JSON representation of a Nix derivation (version 3).
+
+  This schema describes the JSON representation of Nix's `Derivation` type.
+
+  > **Warning**
+  >
+  > This JSON format is currently
+  > [**experimental**](@docroot@/development/experimental-features.md#xp-feature-nix-command)
+  > and subject to change.
+
+type: object
+required:
+  - name
+  - version
+  - outputs
+  - inputSrcs
+  - inputDrvs
+  - system
+  - builder
+  - args
+  - env
+properties:
+  name:
+    type: string
+    description: |
+      The name of the derivation.
+      Used when calculating store paths for the derivation’s outputs.
+
+  version:
+    const: 3
+    description: |
+      Must be `3`.
+      This is a guard that allows us to continue evolving this format.
+      The choice of `3` is fairly arbitrary, but corresponds to this informal version:
+
+      - Version 0: A-Term format
+
+      - Version 1: Original JSON format, with ugly `"r:sha256"` inherited from A-Term format.
+
+      - Version 2: Separate `method` and `hashAlgo` fields in output specs
+
+      - Version 3: Drop store dir from store paths, just include base name.
+
+      Note that while this format is experimental, the maintenance of versions is best-effort, and not promised to identify every change.
+
+  outputs:
+    type: object
+    description: |
+     Information about the output paths of the derivation.
+     This is a JSON object with one member per output, where the key is the output name and the value is a JSON object as described.
```

We could borrow the technical term from the JSON guidelines,

```suggestion
     This JSON object is a dictionary with one member per output, where the key is the output name and the value is a JSON object as described.
```

or use a less explicit but more natural term:

```suggestion
     This JSON object is a mapping with one member per output, where the key is the output name and the value is a JSON object as described.
```